bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -165,7 +165,8 @@ public String getScheme() {
     protected void initialize(AbstractConfiguration<?> conf,
                               StatsLogger statsLogger,
                               RetryPolicy zkRetryPolicy,
-                              Optional<Object> optionalCtx) throws MetadataException {
+                              Optional<Object> optionalCtx,
+                              boolean zkRetryExpired) throws MetadataException {
```

Comment:
this check(zkRetryExpired) is difficult to understand, what scene is set to true and what scene is set to false